django/contrib/auth/locale/pl/LC_MESSAGES/django.po

```diff
@@ -44,13 +44,23 @@ msgstr "Ważne daty"
 msgid "%(name)s object with primary key %(key)r does not exist."
 msgstr "Obiekt %(name)s o kluczu podstawowym %(key)r nie istnieje."
 
+msgid "Conflicting form data submitted. Please try again."
+msgstr "Przesłano konfliktujące dane w formularzu. Proszę spróbować ponownie."
+
 msgid "Password changed successfully."
 msgstr "Hasło zostało zmienione pomyślnie."
 
+msgid "Password-based authentication was disabled."
+msgstr "Uwierzytelnianie oparte na haśle zostało wyłączone."
+
 #, python-format
 msgid "Change password: %s"
 msgstr "Zmień hasło: %s"
 
+#, python-format
+msgid "Set password: %s"
+msgstr " Ustaw hasło: %s"
```

Comment:
Ah my typo :) Fixed in `transifex`, but don't worry you don't need to refetch.  
```suggestion
msgstr "Ustaw hasło: %s"
```

Reply:
I'm happy to refetch, I will also look into the `sr_Latn` file and why the `.mo` does not show as changed.